lib/ansible/cli/__init__.py

```diff
@@ -46,12 +46,12 @@ def check_encoding():
     """Check file system encoding and locale to ensure UTF-8."""
     try:
         locale.setlocale(locale.LC_ALL, '')
-    except locale.Error as e:
+        dummy, encoding = locale.getlocale()
```

Comment:
Should we call this `set_locale` or something similar, instead of `check_encoding`, since this is the only place we're calling `setlocale`?

Reply:
Renamed to `initialize_locale` like it was when it lived in display.